test/functional/tests/fault_injection/test_cache_insert_error.py

```diff
@@ -21,22 +21,29 @@
 from test_utils.os_utils import Udev
 from test_utils.size import Size, Unit
 
+start_size = Size(512, Unit.Byte)
+stop_size = Size(128, Unit.KibiByte)
+
 
-@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.xfail
```

Comment:
Reason for xfail is that for particular blocksize in Write Through mode (128 KB to be precise) was not passing, due to error incrementation occuring more than once in OCF. Fix for this is introduced in https://github.com/Open-CAS/ocf/pull/751

Reply:
That's good to know! However, I meant to put it in xfails reason, like this:
```python
@pytest.mark.xfail(reason="<the_stuff_you_wrote>")
```
So it will get preserved in code for someone who didn't read this PR.

Reply:
On the other hand [Open-CAS/ocf#751](https://github.com/Open-CAS/ocf/pull/751) is already merged, so maybe that's irrelevant at this point anyways :)

Reply:
Heh, and I have pushed already description 😄